MITADS-Speech/lablita_importer.py

```diff
@@ -232,10 +277,18 @@ def save_audio(filename,audio_path,output_dir):
     #with open(file_txt, 'w') as filetowrite:
     #    filetowrite.write(transcription)
 
-    file_mp3 = output_dir + filename + '.mp3'
+    file_mp3 = os.path.join(output_dir,   filename + '.mp3')
     urllib.request.urlretrieve (audio_path, file_mp3)
```

Comment:
There is need a try catch for: TimeoutError: [Errno 110] Connection timed out

Reply:
ok, added try catch